lib/UA.js

```diff
@@ -355,6 +355,9 @@ module.exports = class UA extends EventEmitter
   {
     switch (parameter)
     {
+      case 'authorization_jwt':
+        return this._configuration.authorization_jwt;
+
```

Comment:
Please move it at the end since I don't think this is more relevant than the other password related fields.